tests/e2etest/main.rs

```diff
@@ -647,12 +657,15 @@ async fn http_delete_to_intf(
         )
         .send()
         .await
-        .map_err(|e| format!("HTTP POST failure: {e}"))?
-        .text()
-        .await
-        .map_err(|e| format!("Failure in parsing the HTTP POST result: {e}"))?;
-    if !response.is_empty() {
-        println!("Server response: {response}");
+        .map_err(|e| format!("HTTP DELETE failure: {e}"))?;
+    if response.status() != 204 {
```

Comment:
Can we replace 204 with the Reqwest status code enum?
```suggestion
    if response.status() != StatusCode::NoContent{
```